.gitignore

```diff
@@ -7,3 +7,5 @@ vulture.egg-info/
 .cache/
 .coverage
 .tox/
+.pytest_cache/
+*.swp
```

Comment:
Can you add this ignore in your global .gitignore file?

Reply:
Sure!